build/buildConfig.js

```diff
@@ -1,340 +1,340 @@
-const assign = require('object-assign');
```

Comment:
changes to this file should not be committed unless meaningful

Reply:
Done